### DISCUSSION TITLE
How to get Galaxy Brain Achievement

### DISCUSSION THREAD

**Discussion by MineCuta1107** [Schweinepriester/github-profile-achievements#92] [Q&A]
How do I get the Galaxy Brain Achievement? 
@justforuse.. can you create 1 discussions? i need to get the badge

**Comment by GabrielGomesFer** [comment]
You need to become the marked answer to 2 discussions and you can't mark yourself I think

**Comment by justforuse** [comment]
@MineCuta1107 im here😎

**Comment by sourabmaity** [comment]
@MineCuta1107 im also here 🧠
